domains/home/Welcome.jsx

```diff
@@ -0,0 +1,23 @@
+import React from "react";
+import { Button, Text, View } from "react-native";
+import PropTypes from "prop-types";
+
+
+
+function createGame(playerName, navigation) {
+    navigation.navigate('Game', { playerName })
+}
+export default function Welcome(props) {
```

Comment:
you can destructurate playerName and navigation from props, this way its clearer what value you use
```suggestion
export default function Welcome({ playerName, navigation }) {
```